examples/app-npm-webpack/messages/ru.json

```diff
@@ -0,0 +1,23 @@
+{
+	"ru": {
+		"intro-1": "Используйте Globalize для интернационализиции вашего приложения.",
+		"number-label": "Независимое число",
+		"currency-label": "Независимая валюта",
+		"date-label": "Независимая дата",
+		"relative-time-label": "Независимое относительное время",
+		"message-1": "Пример сообщения со смешанными числами \"{number}\", валютой \"{currency}\", датами \"{date}\", и относительным временем \"{relativeTime}\".",
+		"message-2": [
+			"Пример сообщения с поддержкой множественного числа:",
+			"{count, plural,",
+			"   one {У вас осталась одна задача}",
+			"   two {У вас осталось две задачи}",
+			"   three {У вас осталось три задачи}",
```

Comment:
Looks like there are no `three`, `four`, `five` plural categories, and `two` isn't usually used for Russian: http://cldr.unicode.org/index/cldr-spec/plural-rules